packages/playground/src/components/project-settings/form-member-edit-dialog.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { forwardRef } from 'react'
+import {
+  AlertDialog,
+  AlertDialogContent,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogFooter,
+  AlertDialogCancel,
+  AlertDialogDescription,
+  Select,
+  Button,
+  SelectTrigger,
+  SelectContent,
+  SelectValue,
+  SelectItem,
+  Input,
+  Spacer,
+  Icon,
+  ButtonGroup
+} from '@harnessio/canary'
+import { z } from 'zod'
+import { useForm, SubmitHandler } from 'react-hook-form'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { FormFieldSet, MessageTheme } from '../..'
+
+interface FormEditDialogProps {
+  member: { display_name: string; role: string }
+  onSave: (newRole: string) => void
+  onClose: () => void
+  isSubmitting: boolean
+  submitted: boolean
+}
+
+export const FormEditMemberDialog = forwardRef<HTMLDivElement, FormEditDialogProps>(
```

Comment:
`ref` not needed here, it was added because of a React warning. @jessieyu777 will clean this up and add a screenshot of the warning here for reference purposes.

Reply:
noted for revising in another PR

Reply:
@cjlee01 I've removed this forwardRef in the form but it will be showed this warning. @vardanbansal-harness and I talked about it today, since we don't use forwardRef in this case, so I left this warnings currently.
<img width="925" alt="Screenshot 2024-10-23 at 9 46 23 PM" src="https://github.com/user-attachments/assets/1966760a-f02e-484d-a0f5-47517e36e3d5">